check-libraries.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
i would prefer a JS script over bash ;)

Reply:
Do we still need that script if we publish using a pipeline?